cg/models/lims/sample.py

```diff
@@ -10,64 +10,65 @@
+    data_delivery: Optional[str] = None
+    elution_buffer: Optional[str] = None
+    extraction_method: Optional[str] = None
     family_name: str = "NA"
-    formalin_fixation_time: Optional[str]
-    index: Optional[str]
-    index_number: Optional[str]
-    lab_code: Optional[str]
-    organism: Optional[str]
-    organism_other: Optional[str]
-    original_lab: Optional[str]
-    original_lab_address: Optional[str]
-    pool: Optional[str]
-    post_formalin_fixation_time: Optional[str]
-    pre_processing_method: Optional[str]
-    primer: Optional[str]
+    formalin_fixation_time: Annotated[Optional[str], BeforeValidator(lambda v: str(v))] = None
+    index: Optional[str] = None
+    index_number: Optional[str] = None
+    lab_code: Optional[str] = None
+    organism: Optional[str] = None
+    organism_other: Optional[str] = None
+    original_lab: Optional[str] = None
+    original_lab_address: Optional[str] = None
+    pool: Optional[str] = None
+    post_formalin_fixation_time: Annotated[Optional[str], BeforeValidator(lambda v: str(v))] = None
+    pre_processing_method: Optional[str] = None
+    primer: Optional[str] = None
     priority: str = Priority.standard.name
-    quantity: Optional[str]
-    reference_genome: Optional[str]
-    region: Optional[str]
-    region_code: Optional[str]
+    quantity: Annotated[Optional[str], BeforeValidator(lambda v: str(v))] = None
+    reference_genome: Optional[str] = None
+    region: Optional[str] = None
+    region_code: Optional[str] = None
     require_qc_ok: bool = False
-    rml_plate_name: Optional[str]
-    selection_criteria: Optional[str]
+    rml_plate_name: Optional[str] = None
+    selection_criteria: Optional[str] = None
     sex: Literal["M", "F", "unknown"] = "unknown"
     skip_reception_control: Optional[bool] = None
     source: str = "NA"
-    tissue_block_size: Optional[str]
+    tissue_block_size: Optional[str] = None
     tumour: Optional[bool] = False
-    tumour_purity: Optional[str]
-    volume: Optional[str]
-    well_position_rml: Optional[str]
-    verified_organism: Optional[bool]
+    tumour_purity: Annotated[Optional[str], BeforeValidator(lambda v: str(v))] = None
+    volume: Annotated[Optional[str], BeforeValidator(lambda v: str(v))] = None
+    well_position_rml: Optional[str] = None
+    verified_organism: Optional[bool] = None
 
-    @validator("sex", pre=True)
+    @field_validator("sex", mode="before")
+    @classmethod
     def validate_sex(cls, value: str):
         return SEX_MAP.get(value, "unknown")
 
 
 class LimsSample(BaseModel):
     name: str
     container: str = "Tube"
-    container_name: Optional[str]
-    well_position: Optional[str]
-    index_sequence: Optional[str]
-    udfs: Optional[Udf]
+    container_name: Optional[str] = None
+    well_position: Optional[str] = None
+    index_sequence: Optional[str] = None
+    udfs: Optional[Udf] = None
 
     @classmethod
     def parse_obj(cls, obj: dict):
```

Comment:
Consider a type hint since parse_obj is not the clearest of names (although LimsSample.parse_obj() might be I guess.